frontend/src/metabase/visualizations/components/TableSimple/TableCell.tsx

```diff
@@ -175,8 +176,9 @@ export function TableCell({
         DashboardS.fullscreenNightText,
         EmbedFrameS.fullscreenNightText,
         {
-          "Table-ID": value != null && isID(column),
-          "Table-FK": value != null && isFK(column),
+          [TableS.TableID]: value != null && isID(column),
+          "test-Table-ID": value != null && isID(column),
```

Comment:
maybe instead of fake class names we can use `data-table-id` and `data-table-fk` with their values on `CellRoot`

Reply:
I think the solution you have is ok but I think for the sake of consistency and trying to remove class selectors from cypress I think it's a better direction

Reply:
Yeah that looks nasty. Let's keep it as is then